modulemd/include/private/modulemd-packager-v3-private.h

```diff
@@ -67,28 +67,6 @@ modulemd_packager_v3_to_defaults (ModulemdPackagerV3 *self,
 ModulemdModuleStreamV2 *
 modulemd_packager_v3_to_stream_v2 (ModulemdPackagerV3 *self, GError **error);
 
-/**
- * modulemd_packager_v3_to_stream_v2_ext:
- * @self: (in): This #ModulemdPackagerV3 object.
- * @error: (out): A #GError that will return the reason for a conversion error.
- *
- * Note: If buildopts (#ModulemdBuildopts) are in use in one or more build
- * configurations in the #ModulemdPackagerV3 object @self, only the buildopts
- * present in the first listed configuration (if any) will be applied to the
- * #ModulemdModuleStreamV2 object in the returned index.
- *
- * Returns: (transfer full): A newly-allocated #ModulemdModuleIndex object
- * containing a #ModulemdModuleStreamV2 object and possibly a
- * #ModulemdDefaults object corresponding to the #ModulemdPackagerV3 object
- * @self. NULL if there was an error doing the mapping and sets @error
- * appropriately.
- *
- * Since: 2.11
- */
-ModulemdModuleIndex *
-modulemd_packager_v3_to_stream_v2_ext (ModulemdPackagerV3 *self,
-                                       GError **error);
-
```

Comment:
We will probably still want this function around internally so we can handle the implicit conversion for the `ModuleStream.read_*()` functions.